sdk/communication/AzureCommunicationChat/Source/ChatClient.swift

```diff
@@ -65,6 +68,21 @@ public class ChatClient {
         )
 
         self.service = client.chat
+
+        var token: String?
+
+        credential.token(completionHandler: {(communicationAccessToken, _)
+            in
+            if communicationAccessToken != nil {
+                token = communicationAccessToken!.token
+            }
+        })
+
+        if token != nil {
+            self.signalingClient = getSignalingClient(token: token!)
+        } else {
+            self.signalingClient = nil
+        }
```

Comment:
You should avoid using `!` to force unwrap something. The follow is more swifty:

```suggestion
        if let unwrapped = token {
            self.signalingClient = getSignalingClient(token: unwrapped)
        } else {
            self.signalingClient = nil
        }
```

Reply:
changed to if let instead of force unwrap